### PR TITLE
feat(insights): add alpha badge to insights domain view

### PR DESCRIPTION
**Pull request by DominikB2014**
Adds alpha badge to domain view in the sidebar

### CODE REVIEW

File: static/app/views/insights/pages/settings.ts [hunk 1]
```diff
@@ -1,4 +1,6 @@
+import type {BadgeType} from 'sentry/components/badge/featureBadge';
 import {t} from 'sentry/locale';
 
 export const OVERVIEW_PAGE_TITLE = t('Overview');
 export const DOMAIN_VIEW_BASE_URL = 'performance';
+export const DOMAIN_VIEW_RELEAE_LEVEL: BadgeType = 'alpha';
```
Comment (0Calories) [inline static/app/views/insights/pages/settings.ts:6]:
```suggestion
export const DOMAIN_VIEW_RELEASE_LEVEL: BadgeType = 'alpha';
```

File: static/app/components/sidebar/index.tsx [hunk 2]
```diff
@@ -614,6 +615,8 @@ function Sidebar() {
         id="performance-domains"
         initiallyExpanded={false}
         exact={!shouldAccordionFloat}
+        isAlpha={DOMAIN_VIEW_RELEAE_LEVEL === 'alpha'}
+        isBeta={DOMAIN_VIEW_RELEAE_LEVEL === 'beta'}
```
Comment (0Calories) [inline static/app/components/sidebar/index.tsx:619]:
```suggestion
        isAlpha={DOMAIN_VIEW_RELEASE_LEVEL === 'alpha'}
        isBeta={DOMAIN_VIEW_RELEASE_LEVEL === 'beta'}
```

File: static/app/components/sidebar/index.tsx [hunk 3]
```diff
@@ -71,6 +71,7 @@ import {
   MOBILE_LANDING_SUB_PATH,
   MOBILE_LANDING_TITLE,
 } from 'sentry/views/insights/pages/mobile/settings';
+import {DOMAIN_VIEW_RELEAE_LEVEL} from 'sentry/views/insights/pages/settings';
```
Comment (0Calories) [inline static/app/components/sidebar/index.tsx:74]:
```suggestion
import {DOMAIN_VIEW_RELEASE_LEVEL} from 'sentry/views/insights/pages/settings';
```

**Review by 0Calories — APPROVED** [review]

**Comment by 0Calories** [comment]
image

We should fix the truncation / spacing here before merging this

File: static/app/views/insights/pages/settings.ts (hunk 1 above)
Comment (DominikB2014) [reply static/app/views/insights/pages/settings.ts:6]: Damn, i can't spell lol

**Pull request merged**